modules/text/tutorials/install_tesseract/install_tesseract.markdown

```diff
@@ -0,0 +1,79 @@
+  echo "update $1"
+  cd $1
+  git pull
+  cd ..
+fi
+}
+echo "Installation de leptonica et tesseract"
+myRepo=$(pwd)
+CMAKE_CONFIG_GENERATOR="Visual Studio 14 2015 Win64"
+
+MAJGitRepo leptonica https://github.com/DanBloomberg/leptonica.git
+RepoSource=leptonica
+pushd Build/$RepoSource
+cmake -G"$CMAKE_CONFIG_GENERATOR" -DCMAKE_INSTALL_PREFIX="$myRepo"/Install/leptonica "$myRepo/$RepoSource"
+echo "************************* $Source_DIR -->debug"
+cmake --build .  --config release
+cmake --build $RepoSource  --target install --config release
+popd
+
+RepoSource=tesseract
+MAJGitRepo $RepoSource https://github.com/tesseract-ocr/tesseract.git
+pushd Build/$RepoSource
+cmake -G"$CMAKE_CONFIG_GENERATOR"  -DBUILD_TRAINING_TOOLS:BOOL=OFF -DCMAKE_INSTALL_PREFIX="$myRepo"/Install/tesseract -DLeptonica_DIR:PATH="$myRepo"/Install/leptonica/cmake -DPKG_CONFIG_EXECUTABLE:BOOL=OFF "$myRepo"/"$RepoSource"
+echo "************************* $Source_DIR -->release"
+cmake --build . --config release
+cmake --build .  --target install --config release
+
+popd
+RepoSource=opencv
+pushd Build/$RepoSource
+CMAKE_OPTIONS='-DBUILD_PERF_TESTS:BOOL=OFF -DBUILD_TESTS:BOOL=OFF -DBUILD_DOCS:BOOL=OFF -DWITH_CUDA:BOOL=OFF'
+cmake -G"$CMAKE_CONFIG_GENERATOR"  \
+-DTesseract_INCLUDE_DIR:PATH="${myRepo}"/Install/tesseract/include -DTesseract_LIBRARY="${myRepo}"/Install/tesseract/lib/tesseract400.lib -DLept_LIBRARY="${myRepo}"/Install/leptonica/lib/leptonica-1.74.4.lib \
+$CMAKE_OPTIONS -DOPENCV_EXTRA_MODULES_PATH=../../opencv_contrib/modules \
+-DINSTALL_CREATE_DISTRIB=ON -DCMAKE_INSTALL_PREFIX="$myRepo"/install/"$RepoSource"  "$myRepo/$RepoSource"
+echo "************************* $Source_DIR -->devenv debug"
+cmake --build .  --config debug
+echo "************************* $Source_DIR -->devenv release"
+cmake --build .  --config release
+cmake --build .  --target install --config release
+cmake --build .  --target install --config debug
+popd
+
+@endcode
+    In this script I suppose you use VS 2015 in 64 bits
+@code{.bash}
+CMAKE_CONFIG_GENERATOR="Visual Studio 14 2015 Win64"
+@endcode
+    and leptonica, tesseract will be installed in c:/lib/install
+@code{.bash}
+-DCMAKE_INSTALL_PREFIX="$myRepo"/install/"$RepoSource" "$myRepo/$RepoSource"
+@endcode
+    with no Perf tests, no tests, no doc, no CUDA and no example
+@code{.bash}
+CMAKE_OPTIONS='-DBUILD_PERF_TESTS:BOOL=OFF -DBUILD_TESTS:BOOL=OFF -DBUILD_DOCS:BOOL=OFF  -DWITH_CUDA:BOOL=OFF -DBUILD_EXAMPLES:BOOL=OFF'
+@endcode
+-#  In git command line enter following command :
+@code{.bash}
+./installTesseract.sh
+@endcode
+-# you will have to choose between clone https://github.com/tesseract-ocr/tessdata or only copy language file you need in a folder (example c:\\lib\\install\\tesseract\\tessdata). If you don't want to add a new folder you must copy language file in same folder than your executable
+-# if you created a new folder then you must add to system  a new variable TESSDATA_PREFIX wiih value c:\\lib\\install\\tessdata
```

Comment:
if you created a new folder, then you must add a new variable, TESSDATA_PREFIX with the value c:\\lib\\install\\tessdata to your system's environment